### PR TITLE
Update presentation

### PR DESCRIPTION
**Pull request by qwandor**
Update things to reflect the current state, and reword some parts a bit.

### CODE REVIEW

File: docs/presentation/slides.md [hunk 1]
```diff
@@ -154,26 +148,46 @@
...
 
-  - Mostly Async and threadsafe (talks over raw sockets).
-  - Theoretically cross platform.
+- `btleplug` - "cross-platform jumble"
+  - Theoretically cross platform, but many features not implemented.
+  - Linux implementation talked to kernel directly over raw sockets, bypassing BlueZ daemon.
+      - Requires extra permissions, adds extra bugs.
+      - This has since been changed.
   - Tried switching to this (but gave up after too many panicking threads).
+  - Andrew now trying to fix it and make it async.
```
Comment (alsuren) [inline docs/presentation/slides.md:167]:
```suggestion
  - Andrew now working to make it suit our async.
```

File: docs/presentation/slides.md [hunk 2]
```diff
@@ -234,40 +248,42 @@ The Rust Bluetooth story is a bit sad (all wrappers around BlueZ).
 
 - Andrew is good at separating things into modules (and crates):
```
Comment (alsuren) [inline docs/presentation/slides.md:249]:
```suggestion
- Separating things into modules (and crates) worked well:
```

**Review by alsuren — APPROVED** [review]

File: docs/presentation/slides.md (hunk 2 above)
Comment (qwandor) [reply docs/presentation/slides.md]: Done.

File: docs/presentation/slides.md (hunk 1 above)
Comment (qwandor) [reply docs/presentation/slides.md]: Done.

**Pull request merged**